api/views.py

```diff
@@ -0,0 +1,148 @@
+import datetime
+from datetime import timedelta, datetime
+
+from django.contrib.auth.models import User, Group
+from django.db.models import Sum
+from django.shortcuts import get_object_or_404
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from api.serializers import UserSerializer, GroupSerializer
+from .models import Employee, Income, Outcome, Event
+from .serializers import EmployeeSerializer, EventMiniSerializer, IncomeSerializer, OutcomeSerializer, EventSerializer
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    queryset = User.objects.all().order_by('-id')
+    serializer_class = UserSerializer
+
+
+class GroupViewSet(viewsets.ModelViewSet):
+    queryset = Group.objects.all()
+    serializer_class = GroupSerializer
+
+
+class EmployeeViewSet(viewsets.ModelViewSet):
+    queryset = Employee.objects.all()
+    serializer_class = EmployeeSerializer
+
+    def list(self, request, *args, **kwargs):
```

Comment:
This lacks the brevity, see, this will work the same way:

```
def list(self, request, *args, **kwargs):
	number_of_employees = request.GET.get('number_of_employees')
	employees = Employee.objects.order_by('-salary')

	if number_of_employees:
		# Where's the validation that nuber_of_employees is a valid number?
		employees = employees[:int(number_of_employees)]
            
	serializer = self.get_serializer(employees, many=True)
	return Response(serializer.data)
```

Reply:
done